.github/workflows/ci.yml

```diff
@@ -16,6 +16,8 @@ jobs:
           - ruby:2.5
           - ruby:2.6.0
           - ruby:2.6
+          - ruby:2.7.0
```

Comment:
I'd rather not add this one yet. This is effectively the same as `2.7` just adds to the build time.

Reply:
It's more about be prepared for future patch versions (`2.7.1`, etc.) that usually happens quite fast after initial release :)
Also, build matrix jobs works in parallel, so should not take additional time.